.github/workflows/e2e-main.yaml

```diff
@@ -0,0 +1,77 @@
+#
+# Copyright (C) 2023 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: e2e-tests-main
+
+on: 
+  push:
+    branches: [main]
+    
+  workflow_dispatch:
+    inputs:
+      fork:
```

Comment:
What is the purpose of these parameters ? I don't see them used in the workflow

Reply:
Nice catch! took that form different usage. will update properly.